src/richie/plugins/simple_text_ckeditor/cms_plugins.py

```diff
@@ -0,0 +1,36 @@
+"""
+Simple text CMS plugin
+"""
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+
+from cms.plugin_base import CMSPluginBase
+from cms.plugin_pool import plugin_pool
+
+from .forms import CKEditorPluginForm
+from .models import SimpleText
+
+
+@plugin_pool.register_plugin
+class CKEditorPlugin(CMSPluginBase):
+    """
+    A plugin to add text with CKEditor widget.
+    """
+
+    model = SimpleText
+    form = CKEditorPluginForm
+    name = _("Simple text")
+    render_template = "richie/simple_text_ckeditor/simple_text.html"
+    cache = True
+    module = settings.FUN_PLUGINS_GROUP
+    allow_children = False
+    disable_child_plugins = True
+
+    fieldsets = ((None, {"fields": ["body"]}),)
+
+    def render(self, context, instance, placeholder):
```

Comment:
missing method docstring. How come you don't get an error?

Reply:
dont know, i'm making test-back and lint-back makefile action before each push

Reply:
By the way it seems the behavior since some time because older plugins like section and large_banner does not have docstring on this method too, also some methods like `__str__` or `save` method don't have one.